.phan/config.php

```diff
@@ -366,7 +366,7 @@
         'vendor/php-http',
         'vendor/phan/phan/src/Phan',
         'vendor/phpunit/phpunit/src',
-        'vendor/endclothing/prometheus_client_php/src',
+        'vendor/promphp/prometheus_client_php/src',
```

Comment:
The endclothing version of this library stopped receiving support.  The promphp version is the continued support fork of this library:

https://github.com/endclothing/prometheus_client_php/issues/56